### ISSUE TITLE
Unable to install due to undocumented dependancies

### ISSUE THREAD

**Issue by lewisnyman** [pixelmord/adminx#5]
Hi, I get this error when I try and install through the UI.

```
Unable to install adminx, block.block.adminx_mainnavigation, block.block.adminx_off_canvas_navigation, block.block.adminx_sidebar_navigation, block.block.adminx_socialmenu, block.block.adminx_socialmenu_2 have unmet dependencies.
```

Am I missing something obvious? I guess It's trying to install a block that doesn't exist? Local_tasks_block?

**Comment by pixelmord** [comment]
Yes this configuration is left over from another project this theme was used in, I need to delete those files...

**Comment by pixelmord** [comment]
hopefully fixed with 410b36bfe2d59a390bb7a8683104e3c8f9abf819
did not have time to test yet

**Comment by pixelmord** [comment]
Tested and removed last foreign config with 09014ce1
